CondFormats/CSCObjects/test/testCSCMapping.cpp

```diff
@@ -71,7 +71,8 @@ int testCSCMapping::runIt(const std::string& config){
   edm::AssertHandler ah;
   int rc=0;
   try {
-    edm::EventProcessor proc(config);
+    std::unique_ptr<edm::ParameterSet> pset; // ???????
```

Comment:
@davidlange6 sorry why pset and not edm::getPSetFromConfig(config) as in the next file?

Reply:
ah - thats a bug I think -looking